src/types.ts

```diff
@@ -63,7 +63,7 @@ interface Options {
   i18n?: {
     defaultLanguage?: string
     languages: string[]
-    strategy?: 'suffix' | 'prefix'
+    strategy?: 'suffix' | 'prefix' | 'prefix_except_default'
```

Comment:
Add `en` as default here, no makes sense to use i18n without a default lang (being used or ignored),  doesn't matter the strategy